tests/data/Q4185820-en/output.html

```diff
@@ -0,0 +1,90 @@
+</p>
+<p>
+  In the early 1900s, the castle/tower was still owned by the Gregory family and became part of nearby Coole Estate, home of Lady Augusta Gregory, Yeats’ lifelong friend. On the estate, Coole House, where Lady Gregory lived, was the centre for meetings for the Irish literary group, a group composed of a great number of preeminent figures of the day. Near this tower, in Coole Park, began the Irish Literary Revival.
+</p>
+<p>
+  Thoor Ballylee is also known today as <b>Yeats’ Tower</b>, because in 1916 (or 1917) Yeats purchased the property for the nominal sum of £35 because he was so enchanted with it and especially as it was located in a rural area. From 1921 to 1929, Yeats and his family lived there as it was his monument and symbol: In both aspects, it satisfied his desire for a rooted place in the countryside. The tower retained its original windows in the upper part. Yeats and his architect, Professor William A. Scott, restored the tower for the next two years and installed larger windows in the lower floors.
+</p>
+<p>
+  As he had an affinity for the Irish language, Yeats dropped the term "castle" in naming the property and replaced it with "<i>Thoor</i>" (<i>Túr</i>), the Irish word for "tower"; thus, the place has been known as Thoor Ballylee. For twelve years, Thoor Ballylee was Yeats’ summer home as it was his country retreat. In a letter to a friend, he wrote, "Everything is so beautiful that to go elsewhere is to leave beauty behind." Consequently, it is no wonder that Yeats was inspired and compelled to create literary works at Ballylee such as poems like <i>The Tower</i> and <i>Coole Park and Ballylee</i>.
+</p>
+<p>
+  In 1929, Ballylee was abandoned as the Yeats family moved out and it fell to disuse and ruin.
+</p>
+<p>
+  In 1951, a scene of John Ford's The Quiet Man in which John Wayne and Maureen O'Hara cross a river was shot next to Thoor Ballylee.
+</p>
+<p>
+  Mary Hanley (1914-1979) was the founder of the Kiltartan Society. A native of Carron, County Clare, Hanley founded the society in 1961 to foster interest in the literary history of the district, especially that of Lady Gregory, Edward Martyn and W.B. Yeats. She was responsible for the restoration of Thoor Ballylee (with the aid of Bord Fáilte and the Yeats family). At the time, the Office of Public Works was owner of the property. Hanley persuaded the poet Padraic Colum to open the castle on Sunday 20 June 1965, the centenary of Yeats’s birth, as <i>Yeats Tower</i> to appear as it was when he lived there and refitted as a Yeats museum containing a collection of first editions and items of furniture. The adjoining miller's cottage became a tea room and shop. This was later expanded by a newly constructed building in the back.
+</p>
+<h2 id="Today">
```

Comment:
Until we start using ids, they can also be stripped out to reduce the size. The code that strips them may mention an issue or TODO to remove it when (if) we use ids in TOCs.